src/services/WolfDen.API/Controllers/Department/Department.cs

```diff
@@ -12,10 +14,16 @@ public class Department(IMediator mediator) : ControllerBase
         private readonly IMediator _mediator=mediator;
 
         
-        [HttpPost]
+        [HttpPost("add-department")]
         public async Task<int> AddDepartment([FromBody] AddDepartmentCommand command,CancellationToken cancellationToken)
         {
             return await _mediator.Send(command,cancellationToken);
         }
+
+        [HttpGet("all")]
```

Comment:
Similar to the previous comment, `all` is not required here.
As per norms, GET request denotes fetching the list of items, and GET("{id}") is more specific and implies fetching individual items.

Reply:
this api only returns name and id inorder to assign manager in front end
 [HttpGet("all")] will return all employee details for employee directory